app/build.gradle

```diff
@@ -9,13 +8,17 @@ android {
         applicationId "com.wikaba.ogapp"
         minSdkVersion 9
         targetSdkVersion 21
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 
     sourceSets {
         main {
             java.srcDirs = ['src/main/java-gen', 'src/main/java']
         }
+
+        //make AS and Gradle aware of the active dependencies
+        androidTest {
+            java.srcDirs = ['src/main/java-gen', 'src/test/java', 'src/test/resources']
+        }
```

Comment:
Granted that I'm still very new to Gradle build systems, why should androidTest be using the src/test directory? My understanding is that the src/test directory is meant for local unit tests meant to be run on the host machine, while the src/androidTest directory is meant for unit tests to run on a device. Please feel free to correct me if I have an error in my understanding.